.pre-commit-config.yaml

```diff
@@ -1,7 +1,16 @@
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.2.0
+    rev: v0.2.1
     hooks:
       - id: ruff
+        name: ruff common
         args: ['--fix']
       - id: ruff-format
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.8.0
+    hooks:
+    - id: mypy
+      args: ['--strict', '--follow-imports=skip', '--ignore-missing-imports', '--python-version=3.8']
+      files: '.*pools?\.py'
+      exclude: 'versions|src/tests'
+      additional_dependencies: [types-PyYAML, types-requests, pydantic<2, sqlalchemy]
```

Comment:
I insist we should not make such an ad-hoc type checking a part of the CI. Type checking should be discussed in a separate issue.

Reply:
I removed mypy from pull request